exercises/B-hello-world/exercise.js

```diff
@@ -1 +1,6 @@
 console.log("Hello world");
+console.log("cd exercises/B-hello-world")
```

Comment:
Interesting test! I hope you realised that this will still just print "cd exercises/B-hello-world", even if this command has a very different meaning when you run it in the console, that is, this will not change directory when run inside the console.log as a string.

Reply:
I understand this. I think I was just messing about with random values to see my output when I run them and left this in... but thanks for the pick up and comment.